site/en/docs/android/trusted-web-activity/integration-guide/index.md

```diff
@@ -110,6 +110,7 @@ Add the Trusted Web Activity by inserting an `activity` tag into the `applicatio
 
     <application
         android:allowBackup="true"
+        android:exported="true"
```

Comment:
Thanks for sending the PR.

My understanding from the docs is that `android:exported="true"` should be added to the `activity` tag, not the `application` tag (as in the example [here](https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/core/template_project/app/src/main/AndroidManifest.xml#L104)). Would you mind moving it to the `activity` tag?